gobblin-hive-registration/src/main/java/gobblin/hive/metastore/HiveMetaStoreBasedRegister.java

```diff
@@ -267,6 +271,14 @@ private void addOrAlterPartition(IMetaStoreClient client, Table table, Partition
     }
   }
 
+  private String stringifyPartition(Partition partition) {
+    if (log.isDebugEnabled()) {
```

Comment:
Can't you just add `log.debug` after line 263? Or is this for skipping the Azkaban default log4j properties?